### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by letmejustputthishere**
bump to latest dfx version

### CODE REVIEW

**Review by ericswanson-dfinity — DISMISSED** [review]

**Comment by letmejustputthishere** [comment]
@fxgst or @ericswanson-dfinity can you please create a new release for me 😇

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,6 @@ RUN . "${NVM_DIR}/nvm.sh" && nvm alias default v${NODE_VERSION}
 # Install dfx
 RUN DFXVM_INIT_YES=true sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
 ENV PATH="/root/.local/share/dfx/bin:$PATH"
-ENV DFX_VERSION=
```
Comment (fxgst) [inline Dockerfile:23]: This needs to stay, otherwise `dfxvm` won't let you change versions.

**Review by fxgst — CHANGES_REQUESTED** [review]

**Comment by fxgst** [comment]
Thanks for the PR. Please also bump the version in the Dockerfile in `/azle`

File: Dockerfile (hunk 1 above)
Comment (letmejustputthishere) [reply Dockerfile]: does this mean we have to add it to the azle dockerfile as well?

Comment (fxgst) [reply Dockerfile]: In the azle Dockerfile we just set the env variable at the line where we install dfx, so on the line after it it's not set anymore, whereas here we export it at the top to have all versions at the top, but then we need to unset it at the end of the file to make dfxvm be able to switch versions... I know I should've just used one approach consistently :D

**Review by fxgst — APPROVED** [review]

**Pull request merged**